File: spec/files/scala/bad.scala
object add {
   def bad_function( a:Int, b:Int ) : Int = {

      var i = 0;

      // for loop execution with a range
      for( i <- 1 to 10){
        if( i < 20 ){
            println("This is if statement");
         }
         println( "Value of i: " + i );
      }
      var sum:Int = 0
      sum = a + b
      return sum
   }
}

File: spec/files/GoLang/bad.go
func bad_function() {
	sum := 0
	for i := 0; i < 10; i++ {
		if i := 9; i < 0 {
			fmt.Println(i, "is negative")
			sum += i
		}
	}
	fmt.Println(sum)
	fmt.Println("My favorite number is", rand.Intn(10))
}
